src/utils/convert.js

```diff
@@ -518,11 +580,6 @@ const renderIndex = (legacyApp) => {
           const varName = `${camelCase(name)}${_.capitalize(camelCase(cliType))}`;
           const requireFile = `${dirMap[cliType]}/${snakeCase(name)}`;
           importLines.push(`const ${varName} = require('./${requireFile}');`);
-
-          if (cliType === 'trigger' && _.get(legacyApp, ['general', 'test_trigger_key']) === name) {
-            importLines.push(`const AuthTest = ${varName};`);
-          }
```

Comment:
Test trigger is now imported in `authentication.js`, so we can remove it from `index.js`.